src/Features/Core/Portable/ImplementInterface/AbstractImplementInterfaceService.CodeAction_Method.cs

```diff
@@ -28,9 +28,9 @@ internal partial class ImplementInterfaceCodeAction
                 var updatedMethod = method.EnsureNonConflictingNames(
                     this.State.ClassOrStructType, syntaxFacts, cancellationToken);
 
-                updatedMethod = updatedMethod.RemoveInaccessibleAttributesAndAttributesOfType(
-                    accessibleWithin: this.State.ClassOrStructType,
-                    removeAttributeType: compilation.ComAliasNameAttributeType());
+                updatedMethod = updatedMethod.RemoveInaccessibleAttributesAndAttributesOfTypes(
+                    this.State.ClassOrStructType,
+                    compilation.ComAliasNameAttributeType(), compilation.TupleElementNamesAttributeType());
```

Comment:
We likely do.  However, as .net APIs almost never have dynamic in them, we've likely never noticed.